site/docs/spec.md

```diff
@@ -593,16 +609,17 @@ Table metadata consists of the following fields:
 | _optional_ | _required_ | **`default-spec-id`**| ID of the "current" spec that writers should use by default. |
 | _optional_ | _required_ | **`last-partition-id`**| An integer; the highest assigned partition field ID across all partition specs for the table. This is used to ensure partition fields are always assigned an unused ID when evolving specs. |
 | _optional_ | _optional_ | **`properties`**| A string to string map of table properties. This is used to control settings that affect reading and writing and is not intended to be used for arbitrary metadata. For example, `commit.retry.num-retries` is used to control the number of commit retries. |
-| _optional_ | _optional_ | **`current-snapshot-id`**| `long` ID of the current table snapshot. |
+| _optional_ | _optional_ | **`current-snapshot-id`**| `long` ID of the latest snapshot of the current branch. |
```

Comment:
After the confusion over `current-branch`, I think that the current branch should always be `main` so that people don't try to "checkout" branches in a table. That's always an incorrect operation.

In that case, this should be the `ID of the current table snapshot; must be the same as the current ID of the main branch`.

Reply:
I thought we would eventually have a way for people to switch the `current-branch`, which is like `git checkout branch`. I am not saying we need that from the get go, but why do you think it's incorrect?

Reply:
I think it has proven too confusing to have that feature. Plus, we don't really need it if we support renaming branch refs. Making another branch `main` is as easy as this:

```java
table.updateReferences()
    .rename("main", "main_v1")
    .rename("dev", "main")
    .commit();
```